ee/modules/030-cloud-provider-openstack/images/cloud-data-discoverer/src/go.mod

```diff
@@ -1,15 +1,18 @@
 module discoverer
 
-go 1.23
+go 1.23.1
+
+toolchain go1.24.1
```

Comment:
use .Images.BASE_GOLANG_23_ALPINE
but toolchain 1.24

probably we do not build it in closed env.

and i think we can use new golang image with 1.24

https://github.com/deckhouse/deckhouse/pull/12925